### ISSUE TITLE
Support vendoring

### ISSUE THREAD

**Issue by xclerc** [mirage/ocaml-magic-mime#13]
It looks like the current version cannot be vendored because of
the relative paths in `src/jbuild`. The following patch seems to
fix the issue:

```
diff --git a/src/jbuild b/src/jbuild
index 46299b4..2bfa071 100644
--- a/src/jbuild
+++ b/src/jbuild
@@ -6,7 +6,10 @@
 
 (rule
  ((targets (mime_types.ml))
-  (deps    (../generator/generate_mime_types.exe))
+  (deps    (../generator/generate_mime_types.exe
+            ../mime.types
+            ../x-mime.types
+            ../files.types))
   (action  (with-stdout-to ${@}
-             (progn (run ${<} ../../../mime.types ../../../x-mime.types)
-                    (run ${<} ../../../files.types --files))))))
+             (progn (run ${<} ../mime.types ../x-mime.types)
+                    (run ${<} ../files.types --files))))))
```

**Comment by avsm** [comment]
just noticed this, fixed in #14 as a part of the port to dune and will be in release 1.1.1  (btw, in the future a PR instead of an embedded patch would be welcome if possible)

**Comment by xclerc** [comment]
Sorry for that; I cannot remember why I opened an issue rather than a PR..
